.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
It would be better if you used the `review changes` feature from Github in order to summarise your review👍. Please take a look at this [video](https://www.loom.com/share/e847c46aa4814534b23b3a8e4dd60d6d) to understand how to use it